### PR TITLE
Fixed "don't keep activities" crash on HelpDialog

### LINKED ISSUE #10944: [Bug] Crash: Support AnkiDroid dialog - Don't keep activities
###### Reproduction Steps

1. Have Don't keep activities enabled
2. Open Support AnkiDroid dialog (on the navigation drawer)
3. Minimize the app
4. Open it again

###### Expected Result

No crashes

###### Actual Result

Crash

###### Debug info
```
AnkiDroid Version = 2.16alpha60

Android Version = 12

Manufacturer = samsung

Model = SM-N770F

Hardware = exynos9810

Webview User Agent = Mozilla/5.0 (Linux; Android 12; SM-N770F Build/SP1A.210812.016; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/100.0.4896.88 Mobile Safari/537.36

ACRA UUID = 037bbc86-38a4-4ad7-9c42-05870e9b6086

Scheduler = std2

Crash Reports Enabled = true

DatabaseV2 Enabled = true
```

###### Research
*Enter an [x] character to confirm the points below:*

- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [x] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

### PR DESCRIPTION
**Pull request by dorrin-sot**
## Purpose / Description
App would crash if "Don't keep Activities" was ON and you did as [this video](https://github.com/ankidroid/Anki-Android/issues/10944#issuecomment-1101689732) shows

## Fixes
Fixes #10944 

## Approach
Added `@JvmField` for all `CREATOR`s in `HelpDialog`

## How Has This Been Tested?

https://user-images.githubusercontent.com/59933477/164052808-151a285b-ef92-4c64-a035-ef7ab9403b7a.mov

## Checklist

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Review by mikehardy — APPROVED** [review]
Out of curiosity - this is just a Kotlin question - why not use `const` keyword in Kotlin vs the `@JvmField` style which is more for interoperability vs direct field access (I think, if I understand correctly?)

Note that I literally just read up on JvmField since I did not know exactly how to use it, and the mention that `const` could be used as well was in the Android Developer codelab about interop for JvmField.

Maybe @david-allison has an idea?

This does seem like it will work some I am a soft +1, but I want to know what someone else thinks about the style vs maybe const or maybe something else

**Comment by david-allison** [comment]
> Out of curiosity - this is just a Kotlin question - why not use `const` keyword in Kotlin vs the `@JvmField` style which is more for interoperability vs direct field access (I think, if I understand correctly?)
> 
> Note that I literally just read up on JvmField since I did not know exactly how to use it, and the mention that `const` could be used as well was in the Android Developer codelab about interop for JvmField.

Const only allows primitives and `String`.

File: AnkiDroid/src/main/java/com/ichi2/anki/dialogs/HelpDialog.kt [hunk 1]
```diff
@@ -147,7 +147,7 @@ object HelpDialog {
         private constructor(`in`: Parcel?) : super(`in`!!)
 
         companion object {
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/dialogs/HelpDialog.kt:149]: LGTM. Could you add `@KotlinCleanup("Convert to @Parcelize")` to the classes.

Implementer's choice: implement this KotlinCleanup inside this PR

**Review by david-allison — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/main/java/com/ichi2/anki/dialogs/HelpDialog.kt (hunk 1 above)
Comment (dorrin-sot) [reply AnkiDroid/src/main/java/com/ichi2/anki/dialogs/HelpDialog.kt:149]: OK. Do I just add it to `RateAppItem`, `LinkItem` and `FunctionItem`?

> Implementer's choice: If you have time, implement this KotlinCleanup inside this PR

Sorry I'm really new to Kotlin and don't have enough experience on be able to understand/refactor it myself 😅

Comment (david-allison) [reply AnkiDroid/src/main/java/com/ichi2/anki/dialogs/HelpDialog.kt:151]: Totally fine

**Review by david-allison — APPROVED** [review]

**Pull request merged**